perses/dispersed/utils.py

```diff
@@ -20,6 +20,10 @@
 from scipy.special import logsumexp
 import openmmtools.cache as cache
 
+temperature = 300.0 * unit.kelvin
+kT = kB * temperature
+beta = 1.0/kT
```

Comment:
Why hard-code the temperature? I think this should either be set by the user, or pulled from the htf with introspection if possible. Else, we will run into the issue where someone wants to run at 298K or 310K.

Reply:
We hard the temperature in many perses files, e.g. `relative_point_mutation_setup.py`, 'test_relative.py`, etc

We already have an issue open regarding going through the repo and fixing this: https://github.com/choderalab/perses/issues/810, which I think is something we should address in the long term.

Reply:
Temps are okay in things like tests, I had thought we ran into an issue where we had the temp hard coded and it caused some issues with energy validation, but you're right we never did fix it. It just means that we can't run anything at temp other than 300K. So this is fine for now.